types/fs-cson/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for fs-cson 0.3
+// Project: https://github.com/charlierudolph/fs-cson
+// Definitions by: Piotr Roszatycki <https://github.com/dex4er>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export function readFile(filePath: string, done: (err: Error | null, result: any) => void): void;
+export function readFileSync(filePath: string): any;
+export function register(): any;
```

Comment:
Seems like this returns `void`?

Reply:
I'm not sure. `register()` is defined here  https://github.com/charlierudolph/fs-cson/blob/ea5420ec9001f6abd4f279d83ae2f38187100335/src/register.coffee#L5

and

```
> require('fs-cson').register()
{}
```

and it looks like returns what is returned by `readFileSync`. But I would agree that doesn't make any sense to expect any result from `register` function so I'm going to change it to `void`.